mlctl/clis/batch_cli.py

```diff
@@ -12,24 +12,30 @@ def batch():
 @click.option('--plugin', '-pl', envvar='PLUGIN', help="plugin (i.e. sagemaker, azure)", metavar='', type=click.Choice(['sagemaker']), required=True)
 @click.option('--config', '-c', required=True, help="config file containing batch inference job parameters", metavar='')
 def start(profile, plugin, config):
-    batch = determine_plugin(plugin, profile, 'batch')
-    click.echo(batch.start_batch(config))
+    # TODO: update to include new version of batch
```

Comment:
Are these changes expected? could you please add an explanation?

Reply:
Yes, the old determine_plugin file was removed. I have not yet redone this. This is a good first task